### PR TITLE
Add UI to show subscriber claims

### PR DESCRIPTION
**Pull request by vithu30**
This PR adds UI to show the subscriber claims as shown in the screenshot below:

ui-design

This needs to be merged after merging - https://github.com/wso2/carbon-apimgt/pull/8405

### CODE REVIEW

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/data/api.js [hunk 1]
```diff
@@ -1028,6 +1028,27 @@ class API extends Resource {
...
+                },
+                this._requestMetaData(),
+            );
+        });
+        if (callback) {
+            return promise_subscription.then(callback);
+        } else {
+            return promise_subscription;
+        }
+    }
```
Comment (tmkasun) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/data/api.js:1050]: Please remove the call back usages as below
```suggestion
    getSubscriberInfo(id) {
        const promise_subscription = this.client.then(client => {
            return client.apis['Subscriber'].get_subscriptions__subscriptionId__subscriber_info(
                {
                    subscriptionId: id,
                },
                this._requestMetaData(),
            );
        });
        return promise_subscription;
    }
```

Comment (vithu30) [reply features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/data/api.js]: Fixed

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Subscriptions/SubscriptionsTable.jsx [hunk 2]
```diff
@@ -122,56 +118,24 @@ const styles = (theme) => ({
...
         marginBottom: theme.spacing(3),
     },
+    typography: {
+        padding: theme.spacing(2),
+    },
+    root: {
+        flexGrow: 1,
+    },
+    InfoToolTip: {
+        backgroundColor: '#f5f5f9',
```
[the remark this answers is not in the document]
Comment (vithu30) [reply features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Subscriptions/SubscriptionsTable.jsx]: +1 for leaving the default colors. Changed to default color. Now the view looks as:
Screenshot

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Subscriptions/SubscriptionsTable.jsx [hunk 3]
```diff
@@ -122,56 +118,24 @@ const styles = (theme) => ({
...
+        color: 'rgba(0,0,0,0.87)',
+        maxWidth: 500,
+        fontSize: theme.typography.pxToRem(14),
```
[the remark this answers is not in the document]
Comment (vithu30) [reply features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Subscriptions/SubscriptionsTable.jsx]: Changed to `fontsize` from theme.

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Subscriptions/SubscriptionsTable.jsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (vithu30) [reply features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Subscriptions/SubscriptionsTable.jsx]: Removed the hard-coded values. View:
newview

**Pull request merged**